src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -149,6 +149,12 @@ namespace OpenMS
     */
     void matchModification_(const double mass, const String& origin, String& modification_description);
 
+    /**
+        @brief create a TPP compatible string of the modified sequence using bracket notation
+        i.e.: n[35]RQLNK[162]LQHK[162]GEA
+    */  
+    static String modifiedSequenceToBracketString_(const AASequence & seq);
```

Comment:
I think the proper way to determine the modification state in pepXML should be via the other elements in modification_info. I will check this (probably) on Monday - maybe the parsing is needed for search engine specific files / incomplete pepXML.